src/election-db/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+//! Catalyst Election Database crate
+#![recursion_limit = "256"]
+
```

Comment:
I think we should start with `#![warn(clippy::all)]`, or if we wanna go crazy `clippy::pedantic`, since it's usually more work to add it later.